Controller/QuestionController.php

```diff
@@ -42,6 +42,8 @@ public function showAnswerForm()
     public function answerList()
     {
         $question = QuestionRepository::getQuestionByID($this->request->getGetParameters()['question_id']);
+        if($question->getId()==-1)
+            header("Location: /ask_mvc/question/list");
```

Comment:
Devo averlo già detto da qualche parte  e quindi credo di non aver spiegato bene il funzionamento di **header**: header manda un *header http* ma non interrompe l'esecuzione del codice PHP, che quindi prosegue. In questo caso non cambia niente, ma è comunque non concettualmente corretto, quindi meglio un *if-else*